msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CreateBindingResourcePackageBase.cs

```diff
@@ -34,18 +33,63 @@ public override bool Execute ()
 				return false;
 			}
 
-			string bindingResourcePath = Path.Combine (ProjectDir, OutputPath, Path.ChangeExtension (Path.GetFileName (BindingAssembly), ".resources"));
-			Log.LogMessage (MSBStrings.M0121, bindingResourcePath);
+			var compress = false;
+			if (string.Equals (Compress, "true", StringComparison.OrdinalIgnoreCase)) {
+				compress = true;
+			} else if (string.Equals (Compress, "false", StringComparison.OrdinalIgnoreCase)) {
+				compress = false;
+			} else if (string.Equals (Compress, "auto", StringComparison.OrdinalIgnoreCase)) {
+				compress = ContainsSymlinks (NativeReferences);
+				if (compress)
+					Log.LogMessage (MessageImportance.Low, "Creating a compressed binding resource package because there are symlinks in the input."); // FIXME: localize
```

Comment:
Do these FIXMEs need done before this goes in?

Reply:
Yep, I had big plans of making this a draft. That didn't happen... anyways, fixed now.